src/sidebar/components/test/search-input-test.js

```diff
@@ -84,4 +86,21 @@ describe('SearchInput', () => {
     const wrapper = createSearchInput();
     assert.isTrue(wrapper.exists('button'));
   });
+
+  // FIXME-A11Y
```

Comment:
This is a runtime test that is failing rather than a lint check, so the "FIXME-A11Y" refers to the `it.skip` rather than an `eslint-disable-next-line` comment.